### DISCUSSION TITLE
Quasar extension

### DISCUSSION THREAD

**Discussion by phifuh** [zauberzeug/nicegui#4047] [Q&A]
### Question

I am new to nicegui and i havent figured out how i can use quasar extension. I saw the counter example but that didnt help. 

Can someone provide a minimal code snippen on how to use this component? 

https://mechanicalgux.gitlab.io/quasar-sortable-tree/#/

**Comment by jochso** [comment]
I would like to know this as well. I don't see mentioned in the NiceGUI documentation anywhere?
